core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -32,6 +32,11 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
   ];
   public currentlyHoveredExecution: WorkflowExecutionsEntry | undefined;
 
+  // Pagination attributes
+  public currentPageIndex: number = 1;
+  public pageSize: number = 10;
```

Comment:
maybe make this configurable in [conf](https://github.com/Texera/texera/blob/master/core/new-gui/src/environments/environment.default.ts)